services/ui-src/src/forms/addEditNaaarReport/addEditNaaarReport.json

```diff
@@ -55,7 +55,7 @@
       "validation": "radio",
       "props": {
         "label": "Plan type included in program",
-        "hint": "Indicate the managed care plan type (MCO, PIHP, PAHP, or MMP) that contracts with the state in each program.",
+        "hint": "Indicate the managed care plan type (MCO, PIHP, PAHP, or Other, specify) that contracts with the state in each program.",
```

Comment:
according to the doc this should still have MMP
`Indicate the managed care plan type (MCO, PIHP, PAHP, MMP or Other, specify) that contracts with the state in each program.`